src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
no need to test models, they are considered to be tested implicitly when we test service that use them

Reply:
I completely agree with you, but to achieve the requirement of 80% line coverage, I had to create tests for it.